docs/timezone.md

```diff
@@ -258,17 +261,17 @@ In the case of ambiguity, the `disambiguation` option controls what instant to r
 - `'later'`: The later of two possible times.
 - `'reject'`: Throw a `RangeError` instead.
 
-When interoperating with existing code or services, `'compatible'` mode matches the behavior of legacy `Date` as well as libraries like moment.js, Luxon, and date-fns.
+When interoperating with existing code or services, `'compatible'` mode matches the behavior of legacy `Date` as well as libraries like Moment.js, Luxon, and date-fns.
 This mode also matches the behavior of cross-platform standards like [RFC 5545 (iCalendar)](https://tools.ietf.org/html/rfc5545).
 
-During "skipped" clock time like the hour after DST starts in the Spring, this method interprets invalid times using the pre-transition time zone offset if `'compatible'` or `'later'` is used or the post-transition time zone offset if `'earlier'` is used.
+During "skipped" clock time, like the hour after DST starts in the spring, or when a country permanently changed its UTC offset to one closer to positive infinity, this method interprets invalid times using the pre-transition time zone offset if `'compatible'` or `'later'` is used, or the post-transition time zone offset if `'earlier'` is used.
```

Comment:
> permanently changed its UTC offset to one closer to positive infinity

Could this be said more simply?  Perhaps "larger" or "smaller" ?

Reply:
Yep, I agree with @ptomato-- seems best to leave as-is. Changing the capitalization to "spring" is correct though!  @h-h-h-h you have a great eye for grammar mistakes and typos!